README.md

```diff
@@ -18,9 +18,14 @@ Tambien cuenta con cartas comodines, las cuales son: change_of_color y change_of
 **change_of_color+4**: En caso de que un jugador juegue esta carta se le cambiara de color a los antes mensionado y se le sumará 4 cartas al siguiente jugador. Ojo, solo se puede jugar esta carta en caso de que no tenga mas cartas para jugar.
 
 ###  **Requerimiento:**
-Solo necesitas tener instalado Python, procura no tener una versión muy atrasada.
+Solo necesitas tener instalado Python, procura quue la version sea mayor o igual a la 3.7
 
 ### **Ejecutar**:
-En caso de que hayas descargado todos los archivo,entras todos los archivo dentro del editor(en caso de que tengas algun editor), habrá uno que se llamará: "main" ahí es donde se correrá todo el codigo, solo hay que darle a run("corre") y ahi comenzará el juego.
+En caso de que hayas descargado todos los archivo,entras todos los archivo dentro del editor(en caso de que tengas algun editor), habrá uno que se llamará: "main", desde ahi abres la terminal  que se encuentra en la parte supeior izquierza, colocas el nombre del archivo princpial mas ".py".
+
+En este caso comando sería: "main.py"
```

Comment:
this a tutorial about how to run in a code editor, I need a command to run in bash